backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -230,10 +239,10 @@ export class UserResolver {
     // Save publisherId if Elopage is not yet registered
     if (!user.hasElopage && publisherId) {
       user.publisherId = publisherId
-      await this.updateUserInfos(
-        { publisherId },
-        { sessionId: result.data.session_id, pubKey: result.data.user.public_hex },
-      )
+
+      const loginUser = await LoginUser.findOneOrFail({ email: userEntity.email })
+      loginUser.publisherId = publisherId
+      loginUser.save()
```

Comment:
I am not sure if it has anything to do with the context of this PR!?
I never know when the above line work and when not so:
```suggestion
      const loginUserRepository = getCustomRepository(LoginUserRepository)
      const loginUser = await loginUserRepository.findOneOrFail({ email: userEntity.email })
      loginUser.publisherId = publisherId
      loginUserRepository.save(loginUser)
```